corehq/apps/linked_domain/static/linked_domain/js/domain_links.js

```diff
@@ -87,7 +88,14 @@ hqDefine("linked_domain/js/domain_links", [
 
         // General data
         self.domain = data.domain;
-        self.domain_links = ko.observableArray(_.map(data.linked_domains, DomainLink));
+        self.hasFullAccess = data.has_full_access;
+        self.domainLinks = ko.observableArray(_.map(data.linked_domains, DomainLink));
+        self.domainLinksByNames = ko.computed(function () {
```

Comment:
You could use https://underscorejs.org/#indexBy

Also, I think it's more typical to use the singular: `domainLinksByName`

Reply:
See f803bf426c9230b4d4023fd24e2efbce22eb5778. Thanks for mentioning this cause it pushed me to remove the absolute waste of an observable on the `downstreamDomain` field that I'm sure would have been a head scratcher for some future reader.